ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -363,4 +375,40 @@ public static Schema ensureOptional(final Schema schema) {
         .build();
   }
 
+
+  public static boolean compareSchemaTypes(final Schema arg1, final Schema arg2) {
```

Comment:
```suggestion
  public static boolean areCompatible(final Schema arg1, final Schema arg2) {
```
(I know this was previously my code, but that's a better name 😉 )